tests/testcases/Group10-Vulnerability/10-02-Clair-scan-a-image-with-vulnerability.md

```diff
@@ -0,0 +1,26 @@
+Test 10-02  Clair scan a image with vulnerability
+=======
+  
+# Purpose:  
+To verify that clair can scan a image with vulnerability.  
+
+# References:  
+User guide  
+  
+# Environment:  
+* This test requires that a Harbor instance is running and available.  
+* Harbor is installed with clair enable.  
+* A linux host with Docker CLI installed.  
+  
```

Comment:
Again, an important prereq is:
A prepreq is Clair's Vuln DB has updated successfully.